packages/broker/src/config.ts

```diff
@@ -1,4 +1,4 @@
-export interface TrackerRegistry {
+export interface NetworkSmartContractRegistry {
     registryAddress: string
     jsonRpcProvider: string
 }
```

Comment:
I'd make it even more generic. The nice thing about renaming `registryAddress => contractAddress` is that it matches the name of the protocol util functions that fetch info from smart contracts.
```suggestion
export interface NetworkSmartContract {
    contractAddress: string
    jsonRpcProvider: string
}
```